code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -10,8 +10,11 @@
 	burnmod = 1.25
 	heatmod = 1.5
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/plant
-	disliked_food = NONE
-	liked_food = NONE
+	disliked_food = MEAT
+	liked_food = VEGETABLES | FRUIT
+	toxic_food = TOXIC | RAW
```

Comment:
Citadel just changed it to none in another PR

Reply:
ah i didnt know that, why was it changed?

Reply:
"stealthly nerfed lizrds cannot eat donk pockets" blahblah

Reply:
set to none